pkg/controller/BUILD

```diff
@@ -52,12 +51,10 @@ go_library(
     ],
     importpath = "k8s.io/kubernetes/pkg/controller",
     deps = [
-        "//pkg/api/legacyscheme:go_default_library",
+        "//cmd/controller-manager/app:go_default_library",
```

Comment:
Having anything in //pkg depend on something in //cmd seems like we are reversing the correct direction for dependencies. I realize that //cmd/controller-manager is not a proper //cmd but it still seems backwards. We may want to wait till we have moved //cmd/controller-manager/app out so we are not generating a problematic dependency direction.

Reply:
Thanks for reviewing. Yes, I am waiting for k8s.io/controller-manager be merged first.